src/mapi.ts

```diff
@@ -170,7 +170,15 @@ export class Mapi {
         return message;
     }
 
+    endsWith(str: String, value: String) {
+      return str.slice(-String(value).length) == value;
+    }
+
     searchMapRegExp(url: NormalizedURL): EndpointDetails {
+        // Determines whether url should be treated as a regular expression
+        let rgxToken = "REGEX"
```

Comment:
missing semi colon and use single `let` for multiple declarations. Just like `var`